spec/models/region_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe Region do
+
+  describe 'validations' do
+    it { expect validate_presence_of(:name) }
+    it { expect validate_uniqueness_of(:name).case_insensitive }
```

Comment:
required on AR but optional on postgres, this may lead to data inconsistencies under high concurrency